CMakeLists.txt

```diff
@@ -330,6 +334,7 @@ if (NOT WIN32)
 endif ()
 
 add_library(freeorioncommon
+    $<$<CXX_COMPILER_ID:msvc>:STATIC>
```

Comment:
The repo project files were based on (possibly older) CMake-generated project files.